attacker/types.go

```diff
@@ -11,7 +11,8 @@ type Document struct {
 	RequestTimeout int            `json:"requestTimeout"`
 	MetricName     string         `json:"metricName"`
 	Hostname       string         `json:"hostname"`
-	Rps            int            `json:"rps"`
+	Duration       string         `json:"duration"`
```

Comment:
I do not have any strong opinion on the name. I think its good to keep it as is, because its coming from the config. Regarding its type, I kept it string so that user can specify something like this in their config
```
          - name: OLS_TEST_DURATION
            value: "1m"
```
which we are parsing as duration internally. Changing it to `time.Duration` will make it complicated for the user to specify this argument (.i.e especially in yaml configs)